### ISSUE TITLE
NRCS-NCSS Web of Soils

### ISSUE THREAD

**Issue by ktoddbrown** [ktoddbrown/SoilDRaH#5]
Web of Soil version of NRCS-NCSS https://websoilsurvey.nrcs.usda.gov/app/WebSoilSurvey.aspx

- [x] download data
- [x] annotate data
- [x] shoestring data

**Comment by brandonnodnarb** [comment]
Is this covered off by [soilDB](https://github.com/ncss-tech/soilDB)?

Apologies if this is daft, or has already been covered, but I find the relationship between their data and their outward facing products/services confusing.

**Comment by ktoddbrown** [comment]
@brandonnodnarb Yes, we should link into the package and I also wanted to go through the annotation process with the database as well. I'm aware of this data package but haven't dug into the details here yet. If it links to the backend used to generate the web of science version of the database then the structure is a bit different.

**Comment by ktoddbrown** [comment]
The report and read function is drafted and could use more comments/details.

**Comment by GabrielThrasher** [comment]
More comments/details have been added to the report and read function. **Note**: section 2 (_Soil organic carbon data processing_) of the report file could use more details/comments from someone who fully understands the statistical value/idea that the charts bring/showcase.

**Comment by ktoddbrown** [comment]
@vaasukimarupaka Can you take a look at the NRCS report and read function documentation?
